src/components/content/order/common/BillingModeSelection.tsx

```diff
@@ -0,0 +1,64 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+import React, { Dispatch, SetStateAction } from 'react';
+import { Alert, Flex, Form, Radio } from 'antd';
+import { DeployRequest } from '../../../../xpanse-api/generated';
+
+export const BillingModeSelection = ({
```

Comment:
the billing mode selection is lost when we come back to the  `SelectServiceForm` component.